Makefile

```diff
@@ -231,9 +236,15 @@ copyright:
 	@printf $(COLOR) "Fix license header..."
 	@go run ./cmd/tools/copyright/licensegen.go
 
-lint:
+goimports: update-goimports
```

Comment:
This is already enabled in our golangci-lint config. Why do we need it here as well?

Reply:
The reason is that engineers are ignoring `golangci-lint` errors because valuable errors are lost in the noise.
One option is to make those errors merge blocking. Another option is to make them explicitly visible. I want to make all lint errors merge blocking, but that is a pretty drastic action. This explicit `goimports` is a good short-term solution until we make all lint errors merge blocking. What do you think?

Reply:
Let's disable the noisy alerts or set their severity to "warning". I do see the point that anything which can be auto-fixed should be merge-blocking. I've asked [a question about this in the GitHub action repo](https://github.com/golangci/golangci-lint-action/discussions/683).

We can also do it ourselves in our Makefile or a separate GitHub action using:
```
golangci-lint run --new-from-rev=master --fix=true && [[ -z $(git status -s) ]] || exit 1
```